x-pack/plugins/uptime/server/lib/telemetry/types.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface MonitorUpdateEvent {
+  updatedAt?: string;
+  lastUpdatedAt?: string;
+  durationSinceLastUpdated?: number;
+  deletedAt?: string;
+  type: string;
+  stackVersion: string;
+  monitorNameLength: number;
+  monitorInterval: number;
+  locations: string[];
+  countOfLocations: number;
+  scriptType?: 'inline' | 'recorder' | 'zip';
+  revision: number;
+  error?: Array<{ locationId: string; error: Error }>;
```

Comment:
This type is essentially wrong and will continue to be wrong until the sync errors PR is merged, so I will merge that in first and then resolve this.